nextflow_schema.json

```diff
@@ -32,6 +32,8 @@
                 "input": {
                     "type": "string",
                     "format": "file-path",
+                    "exists": false,
```

Comment:
Don't you want this to be set to true? Or would that interfere with download / building references

Reply:
You are correct, setting it to true interferes with reference building. This is so you do not have to feed in a samplesheet while building, a fake file is used